airflow/utils/db.py

```diff
@@ -1487,6 +1513,12 @@ def upgradedb(
     if errors_seen:
         exit(1)
 
+    if not to_revision and not _get_current_revision(session=session):
+        # Don't load default connections
+        os.environ['AIRFLOW__DATABASE__LOAD_DEFAULT_CONNECTIONS'] = 'False'
```

Comment:
I wonder if this should be an argument to initdb ot something. Setting an environment variable here feels heavy-handed.